layouts/shortcodes/observability_pipelines/log_source_configuration/amazon_s3.md

```diff
@@ -0,0 +1 @@
+REUSE INSTRUCTIONS
```

Comment:
Just flagging this placeholder

Reply:
Ah, this actually isn't needed so will remove.

Reply:
removed